packages/side-nav/src/SideNavGroup/SideNavGroup.tsx

```diff
@@ -121,152 +109,48 @@ function SideNavGroup({
   const accessibleGlyph =
     glyph && isComponentGlyph(glyph)
       ? React.cloneElement(glyph, {
-          className: cx(customIconStyles, glyph.props.className),
+          className: glyph.props.className,
           role: 'presentation',
           'data-testid': 'side-nav-group-header-icon',
         })
       : null;
 
-  // Render the header text
-  const renderedHeader = (
-    <div
-      className={css`
-        display: inline-flex;
-        align-items: center;
-      `}
-    >
-      {accessibleGlyph && (
-        <>
-          {accessibleGlyph}
-
-          <CollapsedSideNavItem active={isActiveGroup}>
-            {accessibleGlyph}
-          </CollapsedSideNavItem>
-        </>
-      )}
-      <Overline
-        className={css`
-          color: inherit;
-        `}
-      >
-        {header}
-      </Overline>
+  const renderHeader = () => (
```

Comment:
Feels like a subcomponent

Reply:
yeah i was going back and forth with this - i'll make it a subcomponent